metersink/api.py

```diff
@@ -20,29 +20,43 @@ def _init_logger():
 
 CONFIG = None
```

Comment:
Same here: Do we need a global here, or can we attach that to the app?

Reply:
the config-object is now settled into the app.config